task/ps_modules/ssrs.psm1

```diff
@@ -511,7 +511,7 @@ function GetJsonFolderItems($Folder, [Folder]$Parent = $null)
 
     foreach($report in $folder.reports)
     {
-        $r = [Report]::new($report.name, $report.fileName, $report.inheritParentSecurity, $report.hidden)
+        $r = [Report]::new($report.name, $report.fileName, $report.inheritParentSecurity, [System.Convert]::ToBoolean($report.hidden))
```

Comment:
In case we make the value of hidden a boolean (check comments above), then there is no need for explicit cast.